packages/cli/src/models/files/ZosNetworkFile.ts

```diff
@@ -59,6 +59,10 @@ interface ProviderInterface {
   address?: string;
 }
 
+interface ProxyAdminInterface {
+  address?: string;
```

Comment:
I'd expect the address to be mandatory, same as with the others. The proxyAdmin object itself should be optional on the package file. I mean, this should be ok:
```json
{
  "app": { "address": "0x1" },
  "proxyAdmin": { "address": "0x2" }
}
```

Same as this:
```json
{
  "app": { "address": "0x1" }
}
```

But not this:
```json
{
  "proxyAdmin": { }
}
```

Reply:
Entirely agree. Will fix it asap.

Reply:
After taking a look at this, I believe we need to define the address as not mandatory because `proxyAdmin`, `app`, `package` and `provider` getters may return an empty object if those address are still not created.